Cargo.toml

```diff
@@ -162,6 +162,12 @@ substrate-frame-rpc-system = { git = "https://github.com/paritytech/polkadot-sdk
 substrate-prometheus-endpoint = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.13.0" }
 sp-debug-derive = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.13.0", default-features = false }
 
+# hotfix for https://github.com/paritytech/polkadot-sdk/issues/7653
```

Comment:
Remove and comment out Cargo CI and create a ticket too fix. Issue should resolve upon upgrade to new Polkadot version.